src/cephadm/cephadm

```diff
@@ -780,9 +784,9 @@ def deploy_daemon(fsid, daemon_type, daemon_id, c, uid, gid,
             fsid, daemon_type, daemon_id,
             uid, gid,
             config, keyring)
-
-    deploy_daemon_units(fsid, uid, gid, daemon_type, daemon_id, c,
-                        osd_fsid=osd_fsid)
+    if not reconfig:
+        deploy_daemon_units(fsid, uid, gid, daemon_type, daemon_id, c,
```

Comment:
Which means we're not restarting or notifying the the daemon at all here?

Reply:
Correct.  For ceph daemons that's fine.  For prom, we need to sighup or something?  @pcuzner?